SmartApi/smartWebSocketOrderUpdate.py

```diff
@@ -0,0 +1,87 @@
+import ssl
+import websocket
+import time
+from logzero import logger
+
+class SmartWebSocketOrderUpdate(object):
+    ROOT_URI = "wss://tns.angelone.in/smart-order-update"
+    HEART_BEAT_MESSAGE = "ping"
+    HEART_BEAT_INTERVAL = 10
```

Comment:
make variables correctly and provide comment

Reply:
make changes in variable name to give meaningfull name